app/models/favorites.php

```diff
@@ -0,0 +1,63 @@
+<?php
+class Favorites extends AppModel
+{
+    public static function vote($user_id, $comment_id, $action)
+    {
+        $db = DB::conn();
+        try {
+            $db->begin();
+            if ($action === 'favorite') {
+                self::insertFavorites($user_id, $comment_id);
+            } else {
+                self::deleteFavorites($user_id, $comment_id);
```

Comment:
Because I use the variable $db for begin(), commit(), and rollback() function po. :bow: Should I remove those functions po?